test/STALK.js

```diff
@@ -24,9 +40,9 @@ describe('STALK', function(done) {
         'value': 5.305800926062761
       });
       delta.should.be.validSignalKDelta;
-      done();
```

Comment:
I had done(); in both the `delta.should..` and `parser.write(...` blocks, but removed the ones in `delta.should....` This is in line with all the other test files.